include/circt/Dialect/FIRRTL/FIRRTLOps.h

```diff
@@ -71,9 +71,10 @@ bool isBundleType(Type type);
 /// connect.
 bool isDuplexValue(Value val);
 
-namespace flow {
-enum Flow { Source, Sink, Duplex };
-}
+enum class Flow { Source, Sink, Duplex };
```

Comment:
This is in the firrtl namespace, but it's a vague term.  Maybe something narrower?